client/app/components/SharedComponents/Card/index.js

```diff
@@ -34,12 +34,13 @@ const SolutionCard = ({ subtitle, title, img, content, url }) => (
   </div>
 );
 
-SolutionCard.propTypes = {
+Card.propTypes = {
   subtitle: PropTypes.string.isRequired,
```

Comment:
Consider removing isRequired, or add subtitles to the topCardContent elements.

![screen shot 2018-01-05 at 8 37 37 am](https://user-images.githubusercontent.com/10067791/34618518-d9810e4a-f1f3-11e7-928e-ad6cb891293f.png)